operator/src/main/java/org/keycloak/operator/crds/v2alpha1/realmimport/KeycloakRealmImportSpec.java

```diff
@@ -29,6 +31,10 @@ public class KeycloakRealmImportSpec {
     @JsonPropertyDescription("The RealmRepresentation to import into Keycloak.")
     private RealmRepresentation realm;
 
+    @JsonProperty("resources")
+    @JsonPropertyDescription("Compute Resources required by Keycloak container")
```

Comment:
I'd add something like:
```suggestion
    @JsonPropertyDescription("Compute Resources required by Keycloak container; if left unspecified, the value is inherited from Keycloak CR")
```